engine/cronjob.go

```diff
@@ -14,7 +14,7 @@ func checkRunningCronjobsConformity(ctx context.Context, l zerolog.Logger, cronj
 	hasBeenPatched := false
 	for _, c := range cronjobs {
 		if *c.Spec.Suspend {
-			l.Info().Str("cronjob", c.Name).Msgf("updating %s from suspend: true to suspend: false", c.Name)
+			l.Info().Str("cronjob", c.Name).Msgf("updating from 'suspend: true' to 'suspend: false'")
```

Comment:
when there is no formatting (i.e no `"%s"` in the string, you should use `.Msg()` instead of `.Msg()`